addon/components/sl-calendar.js

```diff
@@ -127,30 +117,30 @@ export default Ember.Component.extend({
         /**
          * Set the current month and change view mode to that month
          *
-         * @function actions.setMonth
-         * @param    {number} month - The number of the month to change view to
-         * @returns  {void}
+         * @function actions:setMonth
+         * @param {Number} month - The number of the month to change view to
```

Comment:
Do we want a "-" separating the param name and the description? Think this will cause the description to then display as "- ..." in the generated documentation.